marklogic-data-hub-central/build.gradle

```diff
@@ -186,6 +192,7 @@ test {
 	useJUnitPlatform()
 }
 
+tasks.buildRpm.dependsOn(buildUi, copyUiFiles, bootWar)
```

Comment:
Can this line not be in build-rpm.gradle? No problem if it has to be here.

Reply:
Nope, can't move it.